src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/PrisonPersonMoveBookingIntTest.kt

```diff
@@ -284,45 +349,251 @@ class PrisonPersonMoveBookingIntTest : SqsIntegrationTestBase() {
       }
     }
 
+    @Nested
+    inner class Errors {
```

Comment:
Added a bunch of negative tests to prove that the event is retried if any external API calls fail (there are a lot of external API calls!)